odl/contrib/fom/supervised.py

```diff
@@ -129,11 +147,16 @@ def mean_absolute_error(data, ground_truth, mask=None, normalized=False):
     else:
         fom /= l1_norm(data.space.one())
 
+    # The mean absolute error is already in the desired order.
+    # The output does not need to be changed depending on the
+    # force_lower_is_better flag.
+
     return fom
 
 
-def mean_value_difference(data, ground_truth, mask=None, normalized=False):
-    """Return difference in mean value between ``data`` and ``ground_truth``.
+def mean_value_difference(data, ground_truth, mask=None, normalized=False,
```

Comment:
Frankly I'll leave these as is right now to get this out of the way. In general I'm very skeptical to even having this function in ODL since it really makes no sense.